<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-size: 24px; font-weight: bold;
background-color: Yellow;
border-radius: 12px;
border: 2px solid rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Personal Management System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font-size: 24px;
border-radius: 12px;
border: 2px solid rgb(84, 84, 84);
background-color: rgb(0, 85, 255);
color: white;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(220, 220, 220);
border-radius: 12px;
padding: 12px;
font-size: 14px;</string>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(220, 220, 220);
border-radius: 12px;
padding: 12px;
font-size: 14px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="styleSheet">
      <string notr="true">font-size: 16px;
border-radius: 8px;
border: 1px solid gray;
background-color: rgb(0, 85, 255);
color: white;
font-weight: bold;
padding: 10px;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="registerButton">
     <property name="styleSheet">
      <string notr="true">padding: 10px;
font-size: 16px;
border-radius: 8px;
border: 1px solid gray;
background-color: red;;
color: white;
font-weight: bold;
padding: 10px;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
